DXMainClient/DXGUI/Multiplayer/CnCNet/PasswordRequestWindow.cs

```diff
@@ -87,6 +100,7 @@ private void BtnOK_LeftClick(object sender, EventArgs e)
             if (string.IsNullOrEmpty(tbPassword.Text))
                 return;
 
+            pmWindowWasEnabled = false;
```

Comment:
This prevents the private messaging window from being reopened on password window close.